src/platform/site-wide/user-nav/components/PersonalizationDropdown.jsx

```diff
@@ -1,33 +1,14 @@
 import React from 'react';
 
-import { logout } from '../../../user/authentication/utilities';
-import dashboardManifest from '../../../../applications/personalization/dashboard/manifest';
-import recordEvent from '../../../../platform/monitoring/record-event';
-import { mhvBaseUrl } from '../../../../platform/site-wide/cta-widget/helpers';
-import isBrandConsolidationEnabled from '../../../../platform/brand-consolidation/feature-flag';
```

Comment:
This file needed a `rel="noopener noreferrer"` added so I did some cleanup while I was in here.